### DISCUSSION TITLE
blog/caching-with-mediatr-in-aspnet-core/

### DISCUSSION THREAD

**Discussion by giscus** [codewithmukesh/discussions#44] [General]
# blog/caching-with-mediatr-in-aspnet-core/

In this article, we are going to implement Response Caching with MediatR in ASP.NET Core using its awesome Pipeline Behaviours. I have written a couple of articles about MediatR and why it's one of the MUST USE libraries for .NET Developers.

https://codewithmukesh.com/blog/caching-with-mediatr-in-aspnet-core/

**Comment by shrinivas-chakravarthy** [comment]
every article of yours is like a diamond. Keep Going...

**Comment by srsohan150626** [comment]
Superb Article. Keep Providing This Masterpieces .

**Comment by YashSmithh** [comment]
public interface ICacheable
{
    string CacheKey { get; }
    TimeSpan? Expiration { get; }
}

**Comment by TrongNghia-17** [comment]
Absolutely amazing!
